media/js/app/assetmgr/collection.js

```diff
@@ -17,6 +18,37 @@
  * annotation.edit > when edit in place is clicked
  */
 
+// TODO: this renderPage() function really belongs in js/pdf/utils.js,
+// but this file is not a JS module at the moment, making it difficult
+// to import from here.
+const renderPage = function(page, canvas, height) {
```

Comment:
Hmm, I just realized it may be easier than I thought. The key thing is that these modules need to be included with `<script type="module"`. If I can convert just the collection.js file to start, for example, that will make the migration easier. Before, I was under the impression that this would require some big changes to all the files.

Reply:
Nice!